engine/classes/Elgg/Application.php

```diff
@@ -320,6 +320,13 @@ public function __get($name) {
 		trigger_error("Undefined property: " . __CLASS__ . ":\${$name}");
 	}
 
+	/**
+	 *
+	 */
+	public function service($name) {
+		return $this->_services->{$name};
+	}
```

Comment:
I added this just to temporarily bypass the `public_services` property and getting access to all services.